src/files/delete-prompt/DeletePrompt.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import TrashIcon from '../../icons/StrokeTrash'
+import CancelIcon from '../../icons/GlyphSmallCancel'
+import Button from '../../components/button/Button'
+
+const DeletePrompt = ({cancel, action, folder, multiple, className, ...props}) => {
+  className = `${className} w-80 shadow-4 sans-serif relative`
+  let what = 'file'
+  if (folder) {
```

Comment:
@hacdias this needs more thought, what do we do if the user selects 1 folder and 1 file?

Reply:
It should say "files" in my opinion. But what do you think?